student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
+    super(name, age, parent_permission)
```

Comment:
- [ ] Based on feedback on the person class above these two lines should be updated as well. And the name should also be initialized to 'unknown'.

  ```
  def initialize(age, classroom, name = 'unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
      ...
  ```